apps/src/clientApi.js

```diff
@@ -142,11 +161,21 @@ ApiRequestHelper.prototype.delete = function (localUrl, callback) {
 };
 
 /**
- * @param {!string} localUrl
+ * @param {!string} id
+ * @param {!Object} value
+ * @param {!NodeStyleCallback} callback
+ */
+ClientApiRequest.prototype.update = function (id, value, callback) {
+  this.post('/' + id, value, callback);
+};
+
+/**
+ * @param {!string|number} id Id to delete
```

Comment:
The `!` in your JSDoc type here defines `id` as a required parameter (i.e. something you should never pass `undefined` to) but below you have a special case for handling `id === undefined` that appears valid.  Maybe this should actually be an optional parameter?
```
 * @param {string|number} [id] Id to delete
```